sdk/cosmos/azure-cosmos-spark_3_2-12/pom.xml

```diff
@@ -230,6 +232,9 @@
                 <include>com.fasterxml.jackson.core:jackson-databind:[2.13.5]</include> <!-- {x-include-update;com.fasterxml.jackson.core:jackson-databind;external_dependency} -->
                 <include>com.fasterxml.jackson.module:jackson-module-scala_2.12:[2.13.5]</include> <!-- {x-include-update;com.fasterxml.jackson.module:jackson-module-scala_2.12;external_dependency} -->
                 <include>com.fasterxml.jackson.datatype:jackson-datatype-jsr310:[2.13.5]</include> <!-- {x-include-update;com.fasterxml.jackson.datatype:jackson-datatype-jsr310;external_dependency} -->
+                <include>com.fasterxml.jackson.core:jackson-databind:[2.15.2]</include> <!-- {x-include-update;com.fasterxml.jackson.core:jackson-databind;external_dependency} -->
```

Comment:
jackson in two different versions is a slippery slope - let's see whether we can get away with a single version (2.13.5 - or whatever azure-core uses (and is in external_dependency)

Reply:
Only using 2.13.2 now

Reply:
changed back to 2.15.2 for same reason as the other comment